examples/gnrc_border_router/Makefile.wifi.conf

```diff
@@ -0,0 +1,7 @@
+# Set the SSID and password of your WiFi network here
+WIFI_SSID ?= "Your_WiFi_name"
+WIFI_PASS ?= "Your_secure_password"
```

Comment:
Actually, I would prefer to put these variables into the main `Makefile` (as is done e.g. with `ethos`'s config variables, so that a new user sees all the config variables available in one file.

Reply:
Ok, moved it right below the `USE_WIFI` tunable